weblab_cg/_chaste_printer.py

```diff
@@ -0,0 +1,91 @@
+from ._printer import WebLabPrinter
+import sympy
+
+from sympy.printing.precedence import precedence
```

Comment:
Is sympy operator precedence the same as C++ precedence?

Reply:
As far as I can tell for maths expressions they should be no?

Reply:
No idea! There could be small differences :-)
Maybe just check and add a comment above saying they're the same?

Reply:
I would if I could find the sympy precedence  but for the life of me I can't find it.

Reply:
Haha, I suppose it's the same as for Python, but guessing isn't checking :p